addons/sourcemod/scripting/json_test.sp

```diff
@@ -715,13 +735,37 @@ void it_should_concat_arrays()
 
     print_json(arr1);
 
-    Test_AssertEqual("merged length", arr1.Length, 4);
+    Test_AssertEqual("merged length", arr1.Length, 5);
     Test_AssertEqual("merged 2 type", arr1.GetType(2), JSON_Type_Int);
     Test_AssertEqual("merged 2", arr1.GetInt(2), 2);
     Test_AssertEqual("merged 3 type", arr1.GetType(3), JSON_Type_Bool);
     Test_AssertEqual("merged 3", arr1.GetBool(3), false);
     Test_AssertTrue("merged 1 hidden", arr1.GetHidden(1));
     Test_AssertTrue("merged 3 hidden", arr1.GetHidden(3));
```

Comment:
Since the arrays are now not of equal length, this assertion would fail if I hadn't also fixed the hidden key index bug.